homeassistant/components/sleepiq/number.py

```diff
@@ -58,14 +58,14 @@ def _get_actuator_name(bed: SleepIQBed, actuator: SleepIQActuator) -> str:
             f" {bed.name} {actuator.side_full} {actuator.actuator_full} {ENTITY_TYPES[ACTUATOR]}"
         )
 
-    return f"SleepNumber {bed.name} {actuator.actuator_full} {ENTITY_TYPES[ACTUATOR]}"
+    return f"SleepNumber {bed.name} {actuator.actuator_full} {ENTITY_TYPES[ACTUATOR]}"  # type: ignore[unreachable]
 
 
 def _get_actuator_unique_id(bed: SleepIQBed, actuator: SleepIQActuator) -> str:
     if actuator.side:
```

Comment:
`actuator.side` is annotated upstream as a `StrEnum` with non-empty values. Thus it will always be true and the fallback won't be reached. _Not adjusting it here as the mock used in the tests is set to `None` at one point._